src/reducers/index.js

```diff
@@ -54,4 +54,6 @@ export const getActionableFile = ({ files, ui }) => getFileById(files, ui.action
 
 export const mustShowSelectionBar = state => state.ui.showSelectionBar || state.ui.selected.length !== 0
 
+export const isBrowsingTrash = state => state.folder && state.folder.path && state.folder.path.indexOf(TRASH_DIR_PATH) === 0
```

Comment:
I'm not overly fond of th new `TRASH_DIR_PATH` constant, but I really can't see a better way to do this. The main problem is, we only ever know:

- what the current folder's id is
- what the parent's folder id is
- the id of that parent's dir

But if the current folder is 4 levels deep inside the trash, there's no way to find that information. Except in stringified path.